statsnewsletter.php

```diff
@@ -42,6 +46,16 @@ public function __construct()
 		$this->author = 'PrestaShop';
 		$this->need_instance = 0;
 
+		if (version_compare(_PS_VERSION_, '1.7.0.0', '>=')) {
+			$this->table_name = _DB_PREFIX_ . 'emailsubscription';
+			$this->newsletter_module_name = 'ps_emailsubscription';
+			$this->newsletter_module_human_readable_name = 'Email subscription';
```

Comment:
Mmmmmh. You are right.

It's not the way I usually see (like the use of sprintf) but, right right. If the sentence is seen by the PrestaShop Translation tools.